docs/articles/expensify-classic/account-settings/Preferences.md

```diff
@@ -1,5 +1,36 @@
 ---
-title: Preferences
+title: Account-Preferences.md
 description: Preferences
```

Comment:
Please update this description to describe the article. It'll be shown in google search results.

Reply:
Done! I've submitted the update now for the description.